idea/idea-core/src/org/jetbrains/kotlin/idea/core/psiModificationUtils.kt

```diff
@@ -271,10 +269,7 @@ fun KtDeclaration.implicitVisibility(): KtModifierKeywordToken? =
         }
     }
 
-fun KtModifierListOwner.canBePrivate(): Boolean {
-    if (this is KtProperty && this.findAnnotation(FqName("kotlin.jvm.JvmField")) != null) return false
-    return modifierList?.hasModifier(KtTokens.ABSTRACT_KEYWORD) != true
-}
+fun KtModifierListOwner.canBePrivate() = modifierList?.hasModifier(KtTokens.ABSTRACT_KEYWORD) != true
```

Comment:
I'd say it's not the best way to fix it. It's more correct to have all necessary checks inside `canBePrivate` and `canBeInternal` (which does not yet exist). Take into account the fact that both `ChangeVisibilityFix` and `ChangeVisibilityModifierIntention` use `canBePrivate` (and should probably share the same logic). Also I'd recommend to consider content of `canBePrivate` inside `MemberVisibilityCanBePrivateInspection` (private fun). I suspect that some checks from there (not all) could be used inside global `canBePrivate` (that can also fix a bunch of minor bugs, not reported yet).